azdev/config/cli_pylintrc

```diff
@@ -50,13 +53,13 @@ min-similarity-lines=10
 # The invalid-name checker must be **enabled** for these hints to be used.
 include-naming-hint=yes
 
-module-name-hint=lowercase (keep short; underscores are discouraged)
```

Comment:
I am actually curious how `(keep short; underscores are discouraged)` works.

Reply:
This hint is shown in the invalid-name warning. For example:
`demo/demo.py:19:0: C0103: Class name "aa" doesn't conform to '{invalid-classname-hint}' pattern ('[_]{0,2}[A-Z]{1}[A-Za-z0-9]{0,30}$' pattern) (invalid-name)`

I've removed all xxx-hint settings, explainaton: https://github.com/Azure/azure-cli/pull/26685#discussion_r1230661713 and https://github.com/Azure/azure-cli-dev-tools/pull/359#discussion_r1262146487